src/_P036_FrameOLED.ino

```diff
@@ -383,7 +383,7 @@ boolean Plugin_036(uint8_t function, struct EventStruct *event, String& string)
 
         bNoDisplayOnReceivedText = getBitFromUL(PCONFIG_LONG(0), 18);  // Bit 18
         addFormCheckBox(F("Wake display on receiving text"), F("p036_NoDisplay"), !bNoDisplayOnReceivedText);
-        addFormNote(F("Enable this option to not wake-up the display when text is sent from a remote system."));
+        addFormNote(F("Clear this option to not wake-up the display when text is sent from a remote system."));
```

Comment:
"When checked, the display wakes up at receiving remote updates"

Or something like that.
Your change suggests it is always checked, so you're describing how to change it.
A description or note should not describe what will happen if you change it, but it should describe what the state implies.
A user does know checked/unchecked refers to a state, so then you should describe what a state does.
Such descriptions are always more clear when they describe what it does when "enabled" and not when you describe how to turn something off.
So that's why I suggested to have the state of the checkbox to reflect something as "enabled".

Reply:
Agree, apparently I'm in a different zone when coding C++. Appreciate the feedback.

Reply:
Well naming of variables and texts in software is still the most difficult part of programming.

Reply:
Are you expecting more changes on this code?

Reply:
Nope, I will test it tomorrow and then merge it if I don't find anything wrong with it.